make/jdk/src/classes/build/tools/generatelsrequivmaps/EquivMapsGenerator.java

```diff
@@ -263,11 +282,11 @@ private static void generateSourceCode(String fileName) {
                 Paths.get(fileName))) {
             writer.write(getOpenJDKCopyright());
             writer.write(headerText
-                + (int)(sortedLanguageMap1.size() / 0.75f + 1) + ");\n"
-                + "        multiEquivsMap = new HashMap<>("
-                + (int)(sortedLanguageMap2.size() / 0.75f + 1) + ");\n"
-                + "        regionVariantEquivMap = new HashMap<>("
-                + (int)(sortedRegionVariantMap.size() / 0.75f + 1) + ");\n\n"
+                + sortedLanguageMap1.size() + ");\n"
+                + "        multiEquivsMap = HashMap.newHashMap("
+                + sortedLanguageMap2.size() + ");\n"
+                + "        regionVariantEquivMap = HashMap.newHashMap("
+                + sortedRegionVariantMap.size() + ");\n\n"
```

Comment:
It'd be nice if these messy concatenated strings be converted to a text block, but that would be for another day.

Reply:
Will file a separate JBS issue for this